docs/cluster-configuration.md

```diff
@@ -828,3 +801,25 @@ Labels
     annotation_name="kubevirt.kubevirt.io/jsonpatch"
     severity=info
 ```
+## Log verbosity
```

Comment:
Please move up. The HyperConverged formal API should be one after the other. The jsonpatch should remain the last section. Let's put it after the `## Configure custom golden images` section and before the `## Workloads protection on uninstall` section.

Reply:
Done